### PR TITLE
174-add support for key

### LINKED ISSUE #174: Add support for product and variant key
Add support for attribute `key` on product and variant level in export and import process.

Docs: http://dev.commercetools.com/http-api-projects-products.html#product

### PR DESCRIPTION
**Pull request by junajan**
#### Summary
This PR adds a support for `product.key` and `variant.key` attributes.

#### Todo
Tests will currently fail because the old SDK does not know how to generate update action for a `key` attribute.

- Tests
    - [x] Unit
    - [x] Integration
    - [ ] Acceptance
- [x] Documentation

Resolves #174

### CODE REVIEW

**Comment by junajan** [comment]
There has been added PR which implements this feature in the sphere-node-sdk - https://github.com/sphereio/sphere-node-sdk/pull/238

File: src/spec/integration/import.spec.coffee [hunk 1]
```diff
@@ -123,7 +125,7 @@ describe 'Import integration test', ->
         done()
       .catch (err) -> done _.prettify(err)
 
-    it 'should update changes on 2nd import run', (done) ->
+    iit 'should update changes on 2nd import run', (done) ->
```
Comment (wizzy25) [inline src/spec/integration/import.spec.coffee]: `iit` -> `it` so other tests don't get skipped

Comment (junajan) [inline src/spec/integration/import.spec.coffee]: ahhh 🤦‍♂️  .. thx

Comment (junajan) [inline src/spec/integration/import.spec.coffee]: f304f77

**Comment by junajan** [comment]
Hello @hisabimbola, please review this PR so we can merge this.

**Comment by junajan** [comment]
Thx guys

**Pull request merged**